pyro/contrib/examples/util.py

```diff
@@ -2,9 +2,23 @@
 
 import sys
 
+import torchvision
 import torchvision.datasets as datasets
 from torch.utils.data import DataLoader
 from torchvision import transforms
+from torchvision.datasets import MNIST
+
+from pyro.util import patch_dependency
+
+
+@patch_dependency('torchvision.datasets.MNIST', torchvision)
+class _MNIST(MNIST):
```

Comment:
Just to be safe in the presence of Jupyter notebook autoreloads , I'd recommend:
```py
class _MNIST(getattr(MNIST, '_pyro_unpatched', MNIST)):
```